x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table_state.test.tsx

```diff
@@ -653,104 +651,6 @@ describe('AlertsTableState', () => {
     });
   });
 
-  describe('flyout', () => {
```

Comment:
we need to do these test differently, they are still valid in my opinion.

Reply:
I think we just need to test that passing something via the `useActionsColumn` is usable but we shouldn't care what type of action? Let's talk about it this week!